lib/jekyll/converters/scss.rb

```diff
@@ -188,17 +185,14 @@ def sass_configs
         )
       end
 
-      def sass_embedded_config(data)
+      def sass_embedded_config
```

Comment:
I don't remember why this was a public instance method.
Ideally we shouldn't change the signature of public methods. But since this hasn't been released yet, we can overlook the change to arity.